python/fusion_engine_client/parsers/fast_indexer.py

```diff
@@ -0,0 +1,185 @@
+import os
+import math
+from multiprocessing import Pool, cpu_count
+from typing import List, Tuple
+import struct
+
+import numpy as np
+
+from ..messages import MessageHeader, Timestamp, message_type_to_class
+from ..utils import trace as logging
+from .file_index import FileIndex
+
+# NOTE: This needs to be larger then the biggest possible FE message. The
+# smaller it is, the faster the indexer can run, but if it's not large enough,
+# messages may be missed.
+_MAX_FE_MSG_SIZE_BYTES = 1024 * 12
+
+# This has been tuned on my laptop to where increasing the read size was giving
+# diminishing returns on speed.
+_READ_SIZE_BYTES = 80 * 1024
+_READ_WORDS = int(_READ_SIZE_BYTES / 2)
```

Comment:
To be clear, by word in this context I think you mean "2 byte chunk that might have a preamble" and not word in the normal sense of like 32b value, correct? Might suggest renaming to avoid confusion.